src/app.ts

```diff
@@ -42,6 +42,9 @@ const action = async () => {
     core.info(`UTC offset: ${utcOffset}`);
     const exclude = core.getInput('EXCLUDE', {required: false}).split(',');
     core.info(`Excluded languages: ${exclude}`);
+    const isAutoPush = core.getInput('IS_AUTOPUSH', {required: false});
```

Comment:
Just for a reminder, modify the variable here after modifying the `action.yml`.
```ts
const autoPush = core.getInput('AUTO_PUSH', {required: false});
```

Reply:
I think `AUTO_PUSH` is more concise than `IS_AUTO_PUSH` 😄 , we also document the type in `action.yml`, so it's should not be a problem to know its type.